portfolio/src/main/webapp/index.html

```diff
@@ -1217,12 +1217,56 @@ <h2><span>What should we eat?</span></h2>
                     
                     <h2>Fetch Hello Response</h2>
                     <p>Click this button to see the server's response.</p>
-                    <button onclick="showHelloResp();">
+                    <button onclick = "showHelloResp();">
                         Fetch string
                     </button>
-                    <p id="hello-container"></p>
-                    <p>Click <a href="/hello">here</a> to see the server response
+                    <p id = "hello-container"></p>
```

Comment:
Convention is usually no spaces around `=`

Reply:
Thank you for letting me know the convention; I will improve this